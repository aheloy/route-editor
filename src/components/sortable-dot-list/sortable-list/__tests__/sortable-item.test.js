import React from 'react';

import SortableItem from '../sortable-item';


const onDelete = () => 0;


test('render a SortableItem', () => {
  const wrapper = shallow(
    <SortableItem onDelete={onDelete}>
      123
    </SortableItem>
  );
  expect(wrapper).toMatchSnapshot();
});
