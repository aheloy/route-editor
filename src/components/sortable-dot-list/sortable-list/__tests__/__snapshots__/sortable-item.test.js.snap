// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a SortableItem 1`] = `
<SortableItem
  draggable={true}
  onDelete={[Function]}
  onDragEnd={[Function]}
  onDragOver={[Function]}
  onDragStart={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
>
  123
</SortableItem>
`;
